__live-tests__/credit-memo-api.test.ts

```
@@ -0,0 +1,100 @@
+import { AuthProvider, Environment, ApiClient, AuthScopes } from '../src/app';
+import { describe, expect, test } from 'bun:test';
+
+// Describe the CreditMemo API
+describe('Live API: CreditMemos', async () => {
+	// Initialize the Auth Provider
+	const authProvider = new AuthProvider(process.env.QB_CLIENT_ID!, process.env.QB_CLIENT_SECRET!, process.env.REDIRECT_URI!, [
+		AuthScopes.Accounting,
+	]);
+
+	// Deserialize the Token
+	await authProvider.deserializeToken(process.env.SERIALIZED_TOKEN!, process.env.SECRET_KEY!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)